shared/src/commonMain/kotlin/com/innova/innovaPoolingLibrary/res/values/Dimensions.kt

```diff
@@ -0,0 +1,44 @@
+package com.innova.innovaPoolingLibrary.res.values
+
+object Paddings {
+    const val padding1 = 1
+    const val padding2 = 2
+    const val padding3 = 3
+    const val padding4 = 4
+    const val padding5 = 5
+    const val padding6 = 6
+    const val padding7 = 7
+    const val padding8 = 8
+    const val padding9 = 9
+    const val padding10 = 10
+
+    const val padding12 = 12
+    const val padding14 = 14
+    const val padding16 = 16
+    const val padding18 = 18
+    const val padding20 = 20
+    const val padding30 = 30
+
+}
+
+object CornerRadius {
+    const val cornerRadius1 = 1
+    const val cornerRadius2 = 2
+    const val cornerRadius3 = 3
+    const val cornerRadius4 = 4
+    const val cornerRadius5 = 5
+    const val cornerRadius6 = 6
+    const val cornerRadius7 = 7
+    const val cornerRadius8 = 8
+    const val cornerRadius9 = 9
+    const val cornerRadius10 = 10
+
+    const val cornerRadius15 = 15
+    const val cornerRadius20 = 20
+    const val cornerRadius25 = 25
+    const val cornerRadius30 = 30
+    const val cornerRadius35 = 35
+    const val cornerRadius40 = 40
+    const val cornerRadius45 = 45
+    const val cornerRadius50 = 50
```

Comment:
We don't need these many corner radius values. Just keep whatever we using as of now and gradually we can add them here on need basis.